libnavui-dropin/src/main/res/values/dimens.xml

```diff
@@ -3,6 +3,10 @@
     <dimen name="mapbox_cameraModeButton_paddingStart">20dp</dimen>
 
     <!-- This width is mapbox_button_size + padding * 2. mapbox_button_size is 56 -->
-    <dimen name="mapbox_actionList_width">76dp</dimen>
-    <dimen name="mapbox_actionList_padding">10dp</dimen>
+    <dimen name="mapbox_actionList_width">56dp</dimen>
```

Comment:
Will you update the comment above? Seems like we no longer need this `dimen`, or it can reference `dimen/mapbox_button_size`

Reply:
Let's leave the dimen, because we are going to use the new one that Tomasz is implementing in the form of new button designs. @kmadsen